Finjinon/PhotoCaptureViewController.swift

```diff
@@ -143,16 +143,18 @@ open class PhotoCaptureViewController: UIViewController, PhotoCollectionViewLayo
         closeButton.layer.anchorPoint = CGPoint(x: 0.5, y: 0.5)
         containerView.addSubview(closeButton)
 
-        let pickerButtonWidth: CGFloat = 114
-        pickerButton = UIButton(frame: CGRect(x: view.bounds.width - pickerButtonWidth - buttonMargin, y: buttonMargin, width: pickerButtonWidth, height: 38))
-        pickerButton.setTitle(NSLocalizedString("Photos", comment: "Select from Photos buttont itle"), for: UIControlState())
-        pickerButton.setImage(UIImage(named: "PhotosIcon"), for: UIControlState())
-        pickerButton.addTarget(self, action: #selector(presentImagePickerTapped(_:)), for: .touchUpInside)
-        pickerButton.titleLabel?.font = UIFont.preferredFont(forTextStyle: UIFontTextStyle.footnote)
-        pickerButton.autoresizingMask = [.flexibleTopMargin]
-        pickerButton.layer.anchorPoint = CGPoint(x: 0.5, y: 0.5)
-        roundifyButton(pickerButton)
-        view.addSubview(pickerButton)
+        if imagePickerAdapter != nil {
```

Comment:
Potential timing issues? 
i.e. if the imagePickerAdapter is set _after_ the view is loaded, the button will never be created. 

I think we need to either cater for this case or make the limitation abundantly clear in the documentation.